### DISCUSSION TITLE
After the page is zoomed by touch,DevToolClient DispatchTouchEventAsync funcs inject touchpointer has an offset

### DISCUSSION THREAD

**Discussion by flowerlove** [cefsharp/CefSharp#4257] [General]
### Bug Report
Delete this line and everything above, and then fill in the details below.

- **What version of the product are you using?**
 CefSharp 97.1.12

- **What architecture x86 or x64?**
    <x64>
	
- **What version of .Net?**
    <.Net 4.x/>
    
- **On what operating system?**
    <Win10>

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    <WinForms>
    
- **What steps will reproduce the problem?**
   1.use Touch Device to Zoom the page，then use DispatchTouchEventAsync to Inject touchPointer to the browser, but pointer isn't inject on the correct position.

- **What is the expected output? What do you see instead?**
  After zoom the page, DispatchTouchEventAsync should inject the pointer to the correct position.

- **Does this problem also occur in the `CEF` Sample Application**
   also occur

**Comment by amaitland** [comment]
> What version of the product are you using?
CefSharp 97.1.12

Firstly I'd suggest testing with the latest stable version.  Version 97 is very old and unsupported. In future please test with the current supported version before reporting an issue.  

> After zoom the page, DispatchTouchEventAsync should inject the pointer to the correct position.

I suspect this would need to be fixed in chromium. 

> Does this problem also occur in the CEF Sample Application
also occur

If you've managed to reproduce the problem with the CEF Sample application then that basically confirms it's not a CefSharp bug and would need fixing in CEF or Chromium. Based on the description my guess would be it needs fixing in chromium. 

If upgrading doesn't resolve the problem then you'll need to attempt to reproduce the problem with chrome.

**Reply by flowerlove** [reply]
thanks!, I will test the latest stable version.
